### ISSUE TITLE
False positive for "ConditionCouldBeLifted": lambda with parameter

### ISSUE THREAD

**Issue by dimsuz** [appKODE/detekt-rules-compose#31]
This code reports `ConditionCouldBeLifted` while it shouldn't: technically it's legal to have pager with "empty" page.

```kotlin
HorizontalPager(...) { pageIndex ->
  val branchEntry = entries.elementAtOrNull(pageIndex)
  if (branchEntry != null) { // <- reported here
    BranchDetailsWithAction(...)
  }
}
```
